org.js.nuclear.Nuclear.metainfo.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should be upstreamed along with the desktop file & the icon.

Reply:
the metainfo.xml and desktop file are not used by nuclear but are on the upstream repo should anyone want to inspect it. the icon has been taken from upstream.

Reply:
Why not use the files from the upstream tarball then? I don't see a need to copy them over here.

Reply:
the tarball does not have it.

Reply:
You can still use a source of type file and download them from the repo.